### DISCUSSION TITLE
Column width based on content & Horizontal scroll

### DISCUSSION THREAD

**Discussion by premdasvm** [MuhammadRafeh/react-native-datatable-component#15] [Ideas]
Is there any way to extended the column width based on the content? and set horizontal scrolling?

**Comment by MuhammadRafeh** [comment]
Thanks @premdasvm for pointing out. We'll soon implement this feature.

**Comment by MuhammadRafeh** [comment]
@premdasvm Now you can set column width. Please refer to documentation and will soon horizontal scroll will come!
